### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by HarryCusworth**
Have added portfolio images and an icon that replaces them in mobile mode
corrected nasting padding spacing stuff in the side bar
added font support for microsoft computers
changed colours for increased contrast
tidied code

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 				</section>
 		
 				<section class="experience"><div class="contentHeadings">
```
Comment (charliecog) [inline portfolio.html]: div tag to start on its own line

File: portfolio.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
 				<section class="careerProfile">
 					
 					<div class="contentHeadings">
-						<i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> 
+						<i class="fas fa-rocket fa-1.2x" style="color:#CE3B45"></i>
```
Comment (charliecog) [inline portfolio.html]: inline styles are discouraged. Is this a font awesome requirement or can it be moved into your stylesheet?

File: portfolio.html [hunk 3]
```diff
@@ -52,31 +52,57 @@
 							
 				</section>
 		
-				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> Projects</div>
+				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#CE3B45"></i> Projects</div>
```
Comment (charliecog) [inline portfolio.html]: make contentHeadings into contentHeading, more semantic when it represents a singular

File: portfolio.html (hunk 2 above)
Comment (charliecog) [reply portfolio.html]: (assume this comment anywhere you have used an `<i>` tag

File: portfolio.html [hunk 4]
```diff
@@ -52,31 +52,57 @@
...
 				
-						<div class="experienceItemHeading"> Portfolio Project </div>
-						<div class="experienceItemBody"> I did a week's studing and Mayden accademy and all I got was this dumb profile page</div>
 				
+						<div class="portfolioBanner">
+							<div>
```
Comment (charliecog) [inline portfolio.html:60]: remove this div layer, dont need it

File: portfolio.html [hunk 5]
```diff
@@ -52,31 +52,57 @@
...
+								<div class="experienceItemHeading"> Portfolio Project </div>
+						
+								<div class="experienceItemBody"> I did a week's studing and Mayden accademy and all I got was this dumb profile page</div>
+							</div>
+							<div>	
+<!--This div is the portfolio image for display in regular mode - it disapears in mobile mode -->
```
Comment (charliecog) [inline portfolio.html]: remove comments

File: portfolio.html [hunk 6]
```diff
@@ -52,31 +52,57 @@
...
+							</div>
+						</div>
 				</section>
 				
 <!--right hand side of the page content -->
 		</div>
 		<div class="contactBar">
 			<div class="header">
-				<div><img src="images/harryicon.jpeg" class="profilePic" alt="Harry smiling handsomely"></div>
+				<div><img src="images/harryicon.jpeg" class="profilePic" alt="Harry framed and illuminated like a religious icon"></div>
```
Comment (charliecog) [inline portfolio.html]: start new html tag on its own line, also dont forget to add closing slash to the self closing tag eg: ``

**Review by charliecog — COMMENTED** [review]
General comments:
- div class="splitGrid" should just be a `<main>` tag, that way you can get rid of the splitGrid class and just use main tag for the css. Also remove the comment on the line above it.
- your `<link>` that brings in your custom css file should be the last css file in the sequence so that it can override any other styles set elsewhere.
- Use the html validator and fix all the issues that come up to find out syntax changes: https://validator.w3.org/#validate_by_input

File: portfolio.html (hunk 4 above)
Comment (HarryCusworth) [reply portfolio.html:60]: tried to remove, messed up the arrangment a lot though did find one div tag missing a closing slash so maybe that's what threw you

File: portfoliostylesheet.css [hunk 7]
```diff
@@ -71,11 +75,10 @@ h3 {
 	display: flex;
 	align-items: center;
 	flex-direction: column;
-	flex-wrap: nowrap;
-	width: 100%;
 	color: #D0D2CA;
 	padding-top: 40px;
 	padding-bottom: 5px;
+	wrap: wrap;
```
Comment (mporam) [inline portfoliostylesheet.css]: do you mean `flex-wrap`?

File: portfoliostylesheet.css [hunk 8]
```diff
@@ -360,7 +358,7 @@ figcaption {
 	
 	.educationDeets {
 		padding-right: 0px;
-		margin-left: 0%;
+		margin-left: 0px;
 		margin-right: 0px;
 		wrap: wrap;
```
Comment (mporam) [inline portfoliostylesheet.css]: again, i think you mean `flex-wrap`

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**